Sources/Private/Utility/Primitives/BezierPathRoundExtension.swift

```diff
@@ -43,11 +43,8 @@ extension CompoundBezierPath {
 }
 
 extension BezierPath {
-  // Round corners of a single bezier
+  // Computes a new `BezierPath` with each corner rounded based on the given `radius`
   func roundCorners(radius: CGFloat) -> BezierPath {
-    guard radius > 0 else {
```

Comment:
I had to remove this, because it caused artifacts when Core Animation tried to interpolate between a path with `roundedCorners(radius: 0)` to any other value

| Before | After |
| ---- | ---- |
| ![Simulator Screen Shot - iPhone 12 Pro Max - 2022-11-02 at 16 01 44](https://user-images.githubusercontent.com/1811727/199621568-dd613eec-9f41-407c-a44e-25bbc03fd3ed.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2022-11-02 at 16 01 04](https://user-images.githubusercontent.com/1811727/199621572-fff8c443-0fb0-4ddc-a47e-da5ce9d4ed56.png) |

I added back the equivalent logic to the call site from the Main Thread rendering engine, which doesn't have this issue